modules/configuring-huge-pages.adoc

```diff
@@ -92,9 +92,9 @@ $ oc get node <node_using_hugepages> -o jsonpath="{.status.allocatable.hugepages
 ----
 
 ifndef::openshift-origin[]
-[WARNING]
+[NOTE]
 ====
-The TuneD bootloader plugin is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD bootloader plugin is currently not supported.
+The TuneD bootloader plugin is only supported on {op-system-first} worker nodes.
```

Comment:
```suggestion
The TuneD boot loader plugin only supports {op-system-first} worker nodes.
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#boot-loader
The other suggestion is for active voice: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#voice